Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -493,6 +497,31 @@ open class TextStorage: NSTextStorage {
         }
     }
 
+    /// Processes differences in link styles, and applies them to the DOM in the specified
+    /// range.
+    ///
+    /// - Parameters:
+    ///     - range: the range in the DOM where the differences must be applied.
+    ///     - originalStyle: the original link URL object if any.
+    ///     - newStyle: the new link URL object if any.
+    ///
+    private func processLinkDifferences(in range: NSRange, betweenOriginal originalStyle: URL?, andNew newStyle: URL?) {
```

Comment:
Consider renaming the params to better represent the data:

`betweenOriginal originalStyle` -> `betweenOriginal  originalUrl`
`andNew newStyle` -> `andNew newUrl`